Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -887,7 +887,7 @@ static void registerLogHook()
     WebCore::PublicSuffixStore::singleton().enablePublicSuffixCache();
 
 #if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
-    if (isLockdownModeEnabled()) {
```

Comment:
Is there a way to add permanent protection against reintroducing this mistake? Even a comment would be better than nothing, but something stringer would ideal.

Reply:
Ah, good point! I will add a comment, but follow up with a more robust solution.

Thanks for reviewing!